src/leiningen/new/re_frame/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "devDependencies": {
+    "shadow-cljs": "^2.8.36"
+  },
+  "dependencies":{ {{#10x?}}
+    "jquery": "^3.4.0",
+    "react-flip-move": "^3.0.3",
```

Comment:
Are these necessary if you update re-frame-10x to 0.4.0?

Reply:
Turns out jquery was required from an older version of re-pressed 0.3.0. I've updated re-pressed to the latest version. 

It also seems that shadow-cljs is installing react-flip-move and react-highlight.js automatically when they're required so I can remove those as well. From what I can see, version 0.4.0  of re-frame-10x is [still dependant](https://github.com/Day8/re-frame-10x/blob/master/project.clj#L13-L14) on those libs. But I doesn't matter anymore if shadow-cljs is installing them anyway.